### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by msecret**

### CODE REVIEW

File: src/css/base/typography.scss [hunk 1]
```diff
@@ -53,12 +61,10 @@ h5 {
...
 
-body {
-  font-weight: 300;
-  line-height: 1.7333333;
-}
-
 p {
+  font-family: $font-serif;
+  font-size: 1.05rem;
+  line-height: 1.75;
```
Comment (jcscottiii) [inline src/css/base/typography.scss:67]: unit?

File: src/css/base/typography.scss [hunk 2]
```diff
@@ -15,9 +21,11 @@ h1, h2, h3, h4, h5, h6 {
 
 h1 {
   font-size: 2rem;
+  line-height: 1.19;
 
   @include media($medium-screen) {
     font-size: 2.75rem;
+    line-height: 1.18;
```
Comment (jcscottiii) [inline src/css/base/typography.scss:28]: unit?

File: src/css/base/typography.scss [hunk 3]
```diff
@@ -15,9 +21,11 @@ h1, h2, h3, h4, h5, h6 {
 
 h1 {
   font-size: 2rem;
+  line-height: 1.19;
```
Comment (jcscottiii) [inline src/css/base/typography.scss:24]: unit?

File: src/css/base/typography.scss [hunk 4]
```diff
@@ -1,4 +1,10 @@
 
+body {
+  color: $color-base;
+  font-weight: 300;
+  line-height: 1.7333333;
```
Comment (jcscottiii) [inline src/css/base/typography.scss:5]: unit?

File: src/css/base/typography.scss (hunk 1 above)
Comment (msecret) [inline src/css/base/typography.scss:67]: When line-height doesn't have a unit, it makes it relative to the font-size of the current element, which is what I wanted.

**Pull request merged**